### PR TITLE
feature/25_HOME_VIEW

### PR DESCRIPTION
**Pull request by Symphonie234**
- Created a new home view after the user logged in.
- Added new route, css and js files for home view.

### CODE REVIEW

File: app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -114,4 +115,15 @@ protected function registerOrLoginUser($data)
             Auth::login($user);
         }
     }
+
+    public function logout(Request $request)
```
Comment (ChronoDevs) [inline app/Http/Controllers/Auth/LoginController.php:125]: add comment above method

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -13,7 +13,7 @@ class HomeController extends Controller
      */
     public function __construct()
```
Comment (ChronoDevs) [inline app/Http/Controllers/HomeController.php]: remove whole method __construct() if not use

File: routes/web.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\Auth\LoginController;
+use App\Http\Controllers\Auth\LogoutController;
```
Comment (ChronoDevs) [inline routes/web.php]: can't find LogoutController?

File: routes/web.php [hunk 4]
```diff
@@ -16,11 +17,10 @@
 |
 */
 
+// Route localhost to home view
 Route::get('/', function () {
```
Comment (ChronoDevs) [inline routes/web.php]: home route is duplicate.

File: app/Http/Controllers/HomeController.php (hunk 2 above)
Comment (Symphonie234) [reply app/Http/Controllers/HomeController.php]: Understood!

File: routes/web.php (hunk 4 above)
Comment (Symphonie234) [reply routes/web.php]: Understood!

**Comment by Symphonie234** [comment]
- Added comment in the **LoginController** on the logout method.
- Removed the unnecessary use of **LogoutController** namespace in the web.php.
- Removed the duplicate redirection to home in the web.php.

**Pull request merged**